app/screens/LoginScreen.tsx

```diff
@@ -1,28 +1,27 @@
 import { Text, TextInput, TouchableOpacity, View } from "react-native";
 import { useState } from "react";
-import { StyleSheet } from "react-native";
 import { useNavigation } from "@react-navigation/native";
 import { useDispatch } from "react-redux";
 import { setUser } from "@/store/slices/userSlice";
-import auth from "@react-native-firebase/auth";
 import { setAuthenticated } from "@/store/slices/authSlice";
+import { auth } from "@/firebase";
+import { useTheme } from "@/helpers/themeContext";
 
 export const LoginScreen = () => {
   const navigation = useNavigation();
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const dispatch = useDispatch();
+  const { theme } = useTheme();
 
   const handleRedirect = () => {
     navigation.navigate("Register");
```

Comment:
Typescript gives the error here, please refer to [navigation docs](https://reactnavigation.org/docs/typescript/) on how to make type checking